modules/auxiliary/gather/qnap_backtrace_admin_hash.rb

```diff
@@ -90,11 +90,10 @@ def run
 
     if admin_hash
       print_good("Hopefully this is your hash: #{admin_hash}")
-      report_note(
-        host: rhost,
-        port: rport,
-        type: 'qnap.admin.hash',
-        data: admin_hash
+      store_valid_credential(
```

Comment:
@jmartin-r7: So what I'm worried about here is that `service_details` should be set from `HttpClient`, which means `store_valid_credential` should be creating a login object, but this is a non-replayable hash that - if I remember the creds API correctly - should not have a login object associated with it. Can you clarify this for me?